src/coreclr/runtime/portable/WriteBarriers.cpp

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#include <fcall.h>
+
+EXTERN_C FCDECL2(VOID, JIT_WriteBarrier, Object **dst, Object *ref)
```

Comment:
We should not need `JIT_WriteBarrrier`. `JIT_WriteBarrrier` was renamed to `RhpAssignRef` some time ago.